drivers/health_comprehensive_assessment/app/controllers/health_comprehensive_assessment/assessments_controller.rb

```diff
@@ -29,6 +29,7 @@ def edit
 
     def update
       @assessment.update(ca_params)
+      @patient.current_qa_factory.complete_ca(@assessment) if @assessment.completed_on.present?
```

Comment:
Can a completed care plan be edited?  Seems like this might get triggered multiple times for a care plan.

Reply:
This is guarded from duplication.